src/components/popover/bl-popover.stories.mdx

```diff
@@ -0,0 +1,301 @@
+
+Popovers display informative text when users hover over an element.
+
+<bl-alert variant="warning" icon>Inline styles in examples are only for **demo purposes**. Use regular CSS classes or tag selectors to set styles.</bl-alert>
+
+## Basic Usage
+
+Popover can be used with any trigger such as button, icon, text, etc.
+Trigger can be passed to target attribute as element instance or with by id.
+
+<Canvas>
+  <Story name="Show popover with id trigger"  args={{ buttonText:'Trigger by id', id:'1' }} play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+  <Story name="Show popover with instance trigger"  args={{ buttonText:'Trigger with instance' ,id:'2' }} play={popoverOpener}>
+    {TargetInstanceStoryTemplate.bind({})}
+  </Story>
+</Canvas>
+
+## Arrow
+You can give `--bl-popover-arrow-display` for popover arrow visibility. Default arrow is `none`.
+w
+You can use the other options:
+
+* `none`
+* `block`
+
+<Canvas>
+  <Story name="Arrow"
+         args={{ id:'3', style: '--bl-popover-arrow-display: block;--bl-popover-background-color:var(--bl-color-content-tertiary);',
+            buttonText:'Popover with arrow' }}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+</Canvas>
+
+
+## Background Color
+You can give `--bl-popover-background-color` for popover background. Default color is ` #fff`.
+
+<Canvas>
+  <Story name="Background Color"
+         args={{  id:'4', style: '--bl-popover-background-color:var(--bl-color-featured-background);',
+         buttonText:'Custom Background'}}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+</Canvas>
+
+
+## Border Color
+You can give `--bl-popover-border-color` for popover border color. Default color is `#ef6114`.
+
+<Canvas>
+  <Story name="Border Color"
+         args={{  id:'5', style: '--bl-popover-border-color:blue;', buttonText:'Border Color' }}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+</Canvas>
+
+## Padding
+You can give `--bl-popover-padding` for popover padding. Default padding is `1rem`.
+
+<Canvas>
+  <Story name="Padding"
+         args={{  id:'6', style: '--bl-popover-padding:2rem;', buttonText:'Large Padding' }}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+  <Story name="Padding 0"
+         args={{  id:'7', style: '--bl-popover-padding:0;', buttonText:'No Padding' }}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+</Canvas>
+
+## Border Radius
+You can give `--bl-popover-border-radius` for border radius. Default padding is `0.25rem`.
+
+<Canvas>
+  <Story name="Border Radius"
+         args={{  id:'8', style: '--bl-popover-border-radius:0;', buttonText:'Border Radius 0' }}
+         play={popoverOpener}>
+    {StoryTemplate.bind({})}
+  </Story>
+</Canvas>
+
+## Offset
+You can give `offset` for padding from target. Default padding is `10`.
```

Comment:
```suggestion
You can set the distance of the popover to the target element with `offset`. The default distance is `10px`.
```

By the way, are we sure if this default distance is set according to our design system values? I don't think that we have a sizing step as 10px.

@buseselvi what should be the default distance of popover element to its target element?

Reply:
It has 8px padding